library/std/src/sys/windows/process.rs

```diff
@@ -97,6 +98,25 @@ struct DropGuard<'a> {
     lock: &'a Mutex,
 }
 
+pub enum Problem {
+    SawNul,
+    Oversized,
+}
```

Comment:
I'd make the name of the error more descriptive.
```suggestion
pub enum ArgumentError {
    SawNul,
    Oversized,
}
```